casaos-fix-docker-api-version/run.sh

```
@@ -427,7 +442,7 @@ add_user_to_docker_group() {
         echo "OR run this command to start a new shell with updated groups:"
         echo "  newgrp docker"
         echo ""
-        echo "After that, you'll be able to run docker commands without sudo."
+        "After that, you'll be able to run docker commands without sudo."
```

Shellcheck (0.11.0)

[warning] This is interpreted as a command name ending with '.'. Double check syntax.

(SC2288)